doc/user-guides/acrn-shell.rst

```diff
@@ -50,3 +50,179 @@ The ACRN hypervisor shell supports the following commands:
    * - wrmsr [-p<pcpu_id>] <msr_index> <value>
      - Write ``value`` (in hexadecimal) to the Model-Specific Register (MSR) at
        index ``msr_index`` (in hexadecimal) for CPU ID ``pcpu_id``
+
+Command examples
+****************
+
+The following sections provide further details and examples for some of these commands.
+
+vm_list
+=======
+
+"vm_list" provides the name of each virtual machine and its corresponding ID and
+state.
+
+.. figure:: images/shell_image8.png
+   :align: center
+
+   vm_list information
+
+vcpu_list
+=========
+
+"vcpu_list" provides information about virtual CPUs (vCPU), including
+the VM ID, PCPU ID, VCPU ID, VCPU ROLE (primary or secondary), and VCPU
+STATE (init, paused, running, zombie or unknown).
+
+.. figure:: images/shell_image7.png
+   :align: center
+
+   vcpu_list information
+
+vcpu_dumpreg
+============
+
+"vcpu_dumpreg vmid cpuid" provides vCPU related information such as
+registers values, etc.
+
+In the following example, we dump vCPU0 RIP register value and get into
+the SOS to search for the currently running function, using these
+commands::
+
+   cat /proc/kallsyms | grep RIP_value
+
+As you can see, vCPU0 is running in
+function ``acpi_idle_do_entry``.
+
+.. figure:: images/shell_image10.png
+   :align: center
+
+   vcpu_dumpreg information
+
+.. figure:: images/shell_image9.png
+   :align: center
+
+   system map information
+
+dumpmem
+=======
+
+"dumpmem mem_address" provides the specified memory target data such as
+the physical CPU (pCPU) number, etc.
+
+In this example, we know the pCPU active bitmap and physical CPU number
+physical memory address through
+``out/hv/apl-mrb-sbl/hypervisor/acrn.map``. (Note that the path for
```

Comment:
Assuming we can transpose this to a NUC, the right path is: 
```suggestion
``build/hypervisor/acrn.map``. (Note that the path for
```

Reply:
thanks, please david help to update it as geofrroy's comments.